Apparently, some systems put ARG_MAX in a separate development
package that is not always installed.

--- src/monitor_preload.c.orig	2008-06-02 16:59:11.000000000 -0500
+++ src/monitor_preload.c	2008-11-03 18:01:00.000000000 -0600
@@ -58,6 +58,9 @@
 #define MONITOR_EXIT_CLEANUP_SIGNAL  SIGUSR2
 #define MONITOR_POLL_USLEEP_TIME  100000
 #define MONITOR_POLL_MAX 10
+#ifndef ARG_MAX
+#define ARG_MAX  131072
+#endif
 #define MONITOR_MAX_ARG_STRING_SIZE ARG_MAX
 
 /**************************** Forward Declarations **************************/
